### PR TITLE
Add operations

### LINKED ISSUE #3: add operations
- [x] Load from cell (c)
- [x] Load with (c)
- [x] Store (c)
- [x] Move/copy (c)
- [x] Add two's complement (r)
- [ ] Add float (r)
- [x] Or (r)
- [x] And (r)
- [x] Xor (r)
- [x] Rotate (c)

where (r) represents the tasks assigned to me, and (c) represents tasks assigned to @coledn

### PR DESCRIPTION
**Pull request by showalter**
This allows actual operations to be run.

Resolves #3

### CODE REVIEW

File: __main__.py [hunk 1]
```diff
@@ -59,57 +59,106 @@ def create_instruction(a, b):
...
 
 # If instruction looks like 2RXY, load register R
 # with the bit pattern XY
 def load_with(instruction):
-    print("Load with pattern")
 
+    r = registers[int(instruction[1], 16)]
+
+    value = instruction[2:]
+    r.setvalue(value)
```
Comment (showalter) [inline __main__.py]: ensure this is a hex value

File: __main__.py [hunk 2]
```diff
@@ -59,57 +59,106 @@ def create_instruction(a, b):
...
 
 # If instruction looks like 3RXY, store the contents
 # of register R in memory cell XY
 def store(instruction):
-    print("Move")
+    print("Store")
```
Comment (showalter) [inline __main__.py]: remove print statement

File: __main__.py [hunk 3]
```diff
@@ -59,57 +59,106 @@ def create_instruction(a, b):
...
+    xy = cells[int(xy, 16)]
+
+    xy.setvalue(hex(r.getvalue()))
+
 
 
 # If instruction looks like 4*RS, move/copy the bit
 # pattern in register R to register S
 def move(instruction):
     print("Move")
```
Comment (showalter) [inline __main__.py]: remove print statement

File: __main__.py [hunk 4]
```diff
@@ -125,7 +174,6 @@ def rotate(instruction):
 # register R is equal to the bit pattern in register 0
 def jump(instruction):
     global icounter
-    print("jump")
     if instruction[1] == registers[0].getvalue():
```
Comment (showalter) [inline __main__.py]: This should be changed to the register that instruction[1] specifies.

**Review by showalter — COMMENTED** [review]
I can't request changes to my own PR, but here's a few things we should fix.

**Review by showalter — COMMENTED** [review]
I can't approve my own PR, but after looking over everything, it all looks good.

**Review by nuttercd — APPROVED** [review]
Everything looks good!

**Pull request merged**